string msg
---
uint16 box_num
bbox[] bboxes